examples/01_plotting/plot_demo_more_plotting.py

```diff
@@ -134,6 +134,14 @@
                        cut_coords=[36, -27, 60],
                        title="display_mode='tiled'")
 
+########################################
+# Visualizing three views along multiple rows and columns
+# -------------------------------------------------------------------------
+# display_mode='mosaic' for sagittal, coronal and axial view
+
+plotting.plot_stat_map(stat_img, display_mode='mosaic',
+                       title="display_mode='mosaic'")
```

Comment:
Maybe showcase how to use custom cut coordinates here?

Reply:
Done